cpp/include/cudf/dictionary/dictionary_column_view.hpp

```diff
@@ -55,6 +55,12 @@ class dictionary_column_view : private column_view
      */
     column_view indices() const noexcept;
 
+    /**
+     * @brief Returns a column_view combining the indices data
+     * with offset, size, and nulls from the parent.
+     */
+    column_view get_indices_annotated() const noexcept;
```

Comment:
So we have `indices()` (no `get_`), but `get_indices_annotated()`. Is there a good reason to change the naming style here?

Reply:
I've seen conventions where the `get_` is used when the method returns something that is created rather just providing access to existing member data. It could be some other verb like `make` or `build` perhaps. It's always hard to name things of course.

Reply:
ok.